### PR TITLE
MAISTRA-894: Updates for 1.1.0

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: istio-operator/istio-operator.spec [hunk 1]
```diff
@@ -13,7 +13,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit 259bb0321cdabf58218190ef0ac8bf57eeaae43e
+%global git_commit 3a076c29531c848e5d2e846832eb33717f4be042
```
Comment (brian-avery) [inline istio-operator/istio-operator.spec]: should this be ad08ae640ef46cf7a3a25229329e1e02570691a4 ?

File: grafana/003-new-files.patch [hunk 2]
```diff
@@ -853,15 +937,20 @@ index 000000000..85f0dd6aa
...
 +
++%if 0%{?fedora} >= 30
++# Use vendor sources until both Fedora and Grafana properly support golang modules
++%global           unbundle_vendor_sources 0
++%endif
++
 +
 +Name:             grafana
-+Version:          6.2.2
++Version:          6.3.6
```
Comment (brian-avery) [inline grafana/003-new-files.patch:953]: should this be 6.4.3?

Comment (jwendell) [reply grafana/003-new-files.patch:953]: No. Actually it does not matter. This piece of patch changes an internal, not used by us, .spec file, not our spec file.

File: istio-operator/istio-operator.spec (hunk 1 above)
Comment (jwendell) [reply istio-operator/istio-operator.spec]: This pr was issued before an accidental pr on operator got merged. This morning Daniel pushed another one fixing it, that I just picked up.

**Review by brian-avery — APPROVED** [review]

**Review by knrc — APPROVED** [review]

**Review by dgn — APPROVED** [review]

**Review by knrc — APPROVED** [review]

**Pull request merged**